src/it/java/teammates/it/ui/webapi/GetInstructorsActionIT.java

```diff
@@ -0,0 +1,174 @@
+package teammates.it.ui.webapi;
+
+import java.util.List;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.util.Const;
+import teammates.common.util.HibernateUtil;
+import teammates.storage.sqlentity.Instructor;
+import teammates.storage.sqlentity.Student;
+import teammates.ui.output.InstructorData;
+import teammates.ui.output.InstructorsData;
+import teammates.ui.request.Intent;
+import teammates.ui.webapi.GetInstructorsAction;
+import teammates.ui.webapi.JsonResult;
+
+/**
+ * SUT: {@link GetInstructorsAction}.
+ */
+public class GetInstructorsActionIT extends BaseActionIT<GetInstructorsAction> {
+
+    @Override
+    @BeforeMethod
+    protected void setUp() throws Exception {
+        super.setUp();
+        persistDataBundle(typicalBundle);
+        HibernateUtil.flushSession();
+    }
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.INSTRUCTORS;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    @Test
+    @Override
+    protected void testExecute() throws Exception {
+        Instructor instructor = typicalBundle.instructors.get("instructor1OfCourse1");
+
+        loginAsInstructor(instructor.getGoogleId());
+
+        ______TS("Typical Success Case with FULL_DETAIL");
+        String[] params = new String[] {
+                Const.ParamsNames.COURSE_ID, instructor.getCourseId(),
+                Const.ParamsNames.INTENT, Intent.FULL_DETAIL.toString(),
+        };
+
+        GetInstructorsAction action = getAction(params);
+        JsonResult jsonResult = getJsonResult(action);
+
+        InstructorsData output = (InstructorsData) jsonResult.getOutput();
+        List<InstructorData> instructors = output.getInstructors();
+
+        // TODO: Integrate sqlLogic so we can getInstructorsForCourse.
```

Comment:
it's already there. more importantly, there shouldn't be a need to call this at all. you are testing the action not the logic method.